app/controllers/tag_adjustments_controller.rb

```diff
@@ -1,4 +1,8 @@
 class TagAdjustmentsController < ApplicationController
+  after_action only: %i[create destroy] do
+    Audit::Logger.log(:moderator, current_user, params.dup)
```

Comment:
shold we keep the `current_user.auditable?` check here?

Reply:
Same as above!